modules/photons_control/addon.py

```diff
@@ -5,6 +5,7 @@
 from delfick_project.addons import addon_hook
 from delfick_project.norms import sb, Meta
 import binascii
+import ipaddress
```

Comment:
smallest of nits, can you put the ipaddress import above the binascii import. Seems how I do imports is in private photons contributing.rst but not public, I'll add that later.

Essentially,

I group import blocks into groups and then by length of line within the group.

    <imports from to the current module>

    <imports from photons_app>

    <imports from other photons modules>

    <other imports>

For example, inside ``photons_socket``:

    from photons_socket.connection import Sockets

    from photons_app.errors import TimedOut, FoundNoDevices

    from photons_transport.base import TransportItem, TransportBridge, TransportTarget
    from photons_messages import DiscoveryMessages, Services
    from photons_protocol.messages import Messages

    from delfick_project.norms import dictobj, sb
    import logging

Reply:
You know, I knew most of that but I thought the other imports were alphabetical, not by length of line. :)

Reply:
yeah, it's not conventional. But my eyes are bad and it's easier for me to read them when sorted by length :)

Reply:
I'm switching the sorting to IP address for exactly the same reason, so it makes sense to me.

Reply:
hehhehehe